app/screens/home/profile_screen.tsx

```diff
@@ -117,15 +152,16 @@ export default function ProfileScreen({
             textColorType="white"
             darkColor="transparent"
             lightColor="transparent"
-            onPress={() => navigation.navigate("WelcomeFinal")}
+            testID="sign-in-button"
+            onPress={() => logOut(navigation)}
           />
         </ThemedView>
       )}
     </ThemedView>
   );
 }
 
-const styles = {
+const styles = StyleSheet.create({
```

Comment:
Thank you for finding this bug! It was giving me headaches.

Reply:
Haha i get you it got me quite suspicious.